src/mongoose_graphql/admin/mongoose_graphql_admin_mutation.erl

```diff
@@ -0,0 +1,37 @@
+-module(mongoose_graphql_admin_mutation).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+
+execute(_Ctx, _Obj, <<"addDomain">>, #{<<"domain">> := Domain, <<"hostType">> := HostType}) ->
+    case mongoose_domain_api:insert_domain(Domain, HostType) of
```

Comment:
there are a lot of uncovered lines (i.e. error cases are not covered).
And a success case for some operations is not covered too.

Reply:
I was thinking about covering this in another PR dedicated to finishing domain management API